src/main/java/com/revealprecision/revealserver/constants/FormConstants.java

```diff
@@ -0,0 +1,17 @@
+package com.revealprecision.revealserver.constants;
+
+public class FormConstants {
+  public static final String CDD_SUPERVISOR_DAILY_SUMMARY_HEALTH_WORKER_SUPERVISOR_FIELD = "health_worker_supervisor";
```

Comment:
if this class will be used to hold constants, a suggestion would be to just change from class to interface and then wont have  to specify static final